swim/ping_handler.go

```diff
@@ -40,8 +42,11 @@ func handlePing(node *Node, req *ping) (*ping, error) {
 	changes, fullSync :=
 		node.disseminator.IssueAsReceiver(req.Source, req.SourceIncarnation, req.Checksum)
 
+	// Bidirectional full sync.
 	if fullSync {
-		// TODO: handle full sync
+		go func() {
```

Comment:
This seems wasteful to me. If the sender applies the changes from the receiver and their checksums match, a bidirectional full-sync wouldn't be needed. Now what we've got is bi-directional full-syncs by default.

Also, the unbounded go-routine creation frightens me a bit. I know they're meant to be lightweight, but not handling this by throttling the amount of bi-directional full-syncs that we issue seems like something that would come back to haunt us, at scale.